History.md

```diff
@@ -1,6 +1,8 @@
 unreleased
 ==========
 
+  * deps: cookie@0.4.1
```

Comment:
Sort the `deps` section to the bottom of the list (see the other version entries for example).